master/getting-started/mesos/tutorials/launching-tasks.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Launching Tasks
+---
+
+The following information describes how to launch Calico networked tasks in Mesos
+using Marathon.
+
+### Unified Containerizer
+
+Launch a Unified Containerizer task onto a Calico network by setting
+ `networkName` to the name of your Calico network. Its value should match the `"name"`
+ field of the `calico.conf` you configured when [Installing Calico for Mesos](../installation/integration)
+
+```json
+{
+  "id": "unified-task",
+  "cmd": "/usr/sbin/ip addr show && sleep 1000",
+  "ipAddress": {
+      "networkName": "calico"
+  }
+}
+```
+
+>Be sure to replace `/usr/sbin/ip` with the correct path to your IP binary if you are following along.
+
+The task's stdout output should show a Calico IP from the default Calico pool of `192.168.0.0/16`.
+
+### Docker Containerizer
+
+Calico for Docker uses a Profile based network security model which has
+similarly secure tasks on the same network.
+
+To launch a Docker Containerizer task, first create a Docker Network to launch it on
+(or alternatively, identify an existing network with rules for this
+task):
+
+```
+docker network create --driver=calico --ipam-driver=calico calico-nginx
+```
+
+>Alternatively, identify and use an existing network that matches
+
+Then in your marathon application definition,
+set `network` to `USER`, and set `networkName` to the name of your Calico network:
+
+```json
+{
+  "id": "docker-task",
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "nginx",
+      "network": "USER"
+    }
+  },
+  "ipAddress": {
+      "networkName": "calico-nginx"
+  }
+}
+```
+
+## Enabling Healthchecks
+
+Marathon supports Healthchecks for IP per container applications. Healthchecks
+should specify a `port` (instead of the `portIndex` field which is commonly used
+for port-mapped applications).
+
+For the Healthcheck to succeed, the following conditions must be met:
+
+1. The host running Marathon will need routes to Calico tasks. If you are running
+Marathon as a Mesos task, and have already installed Calico on each Agent,
+you have met this requirement.
+
+2. Calico Networking Policy should permit the healthcheck from Marathon to the
+target application.
+
+The following sample application launches a nginx webserver with healtchecks:
+
+```json
+{
+  "id": "nginx",
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "nginx",
+      "network": "USER"
+    }
+  },
+  "ipAddress": {
+      "networkName": "calico"
+  },
+  "healthChecks": [{
+      "protocol": "HTTP",
+      "path": "/",
+      "port": 80
+  }]
+}
+```
+
+The following Calico Profile yaml will allow the healthcheck from an instance
```

Comment:
When would you use this and how?  No mention of running `calicoctl`.

Is this for CNI or Docker or both?